app/src/lib/menu-update.ts

```diff
@@ -204,7 +205,11 @@ function getRepositoryMenuBuilder(state: IAppState): MenuStateBuilder {
     const { conflictState, workingDirectory } = selectedState.state.changesState
 
     rebaseInProgress = conflictState !== null && conflictState.kind === 'rebase'
-    hasConflicts = changesState.conflictState !== null
+    hasConflicts =
+      changesState.conflictState !== null ||
+      workingDirectory.files.some(
+        x => x.status.kind === AppFileStatusKind.Conflicted
+      )
```

Comment:
```suggestion
      hasConflictedFiles(workingDirectory)
```
i think this could use this helper (which does the same thing) https://github.com/desktop/desktop/blob/6287ef907070a6a205f241a8066b3f5e1f71e278/app/src/lib/status.ts#L62-L66